src/styles/RepositoryAvatar.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+import {borderRadius} from "./variables";
+
```

Comment:
```suggestion
import styled from "styled-components";
import {borderRadius, margin, size} from "./variables";
```

Import this to use `gutter` and other sizes

Reply:
Done.